bin/run-wer-automation.sh

```diff
@@ -5,6 +5,9 @@ set -xe
 ds_dataset_path="/data/LIUM/"
 export ds_dataset_path
 
+ds_gpu_usage_root="/data/automation/gpu/"
```

Comment:
And the ds_dataset_path just above has trailing slash :). I prefer having it in the shell script, it makes it easier to track (for me), more obvious, and less error prone.